Assets/Developers/Dennis/Components/Tags/PlayerTagComponent.cs

```diff
@@ -0,0 +1,7 @@
+﻿using UnityEngine;
+
+namespace TagComponents
+{
+    // Meant to be used by the single player in the game for easy identificaion
+    public class PlayerTag : MonoBehaviour { }
```

Comment:
If we're using the hybrid approach, we should use the built-in tag system (Where you set tag and findGameObjectWithTag), since we'll probably attach these monobehavior into a gameobject anyhow.

Reply:
Yeah, was wondering if I should keep it